assets/components/checkboxInput/checkboxInput.jsx

```diff
@@ -11,32 +11,44 @@ type PropTypes = {
   id: ?string,
   onChange: (preference: boolean) => void,
   checked: boolean,
-  labelText?: string,
+  labelTitle?: string,
+  labelCopy?: string,
 };
 
 
 // ----- Component ----- //
 
 export default function CheckboxInput(props: PropTypes) {
-  let label = '';
-
-  if (props.labelText && props.id) {
-    // eslint-disable-next-line jsx-a11y/label-has-for
-    label = <label htmlFor={props.id}>{props.labelText}</label>;
+  let labelTitle = '';
+  let labelCopy = '';
+
+  if (props.id) {
+    if (props.labelTitle) {
+      // eslint-disable-next-line jsx-a11y/label-has-for
+      labelTitle = <p className="component-checkbox__title">{props.labelTitle}</p>;
+    }
+    if (props.labelCopy) {
+      // eslint-disable-next-line jsx-a11y/label-has-for
+      labelCopy = <p className="component-checkbox__copy" >{props.labelCopy}</p>;
+    }
   }
+
   return (
-    <div className="component-checkbox">
+    <label htmlFor={props.id} className="component-checkbox">
       <input
+        className="component-checkbox__checkbox"
         id={props.id}
         type="checkbox"
         onChange={e => props.onChange(e.target.checked)}
         checked={props.checked}
       />
-      {label}
-    </div>
+      {labelTitle}
```

Comment:
personally I find [this style](https://reactjs.org/docs/conditional-rendering.html#inline-if-with-logical--operator
) more readable for conditionally embedding elements in a React component, mainly because you can see the whole structure of the HTML at a glance without having to look elsewhere to see what the variables might contain

Reply:
I'm not sure I agree. I find this cleaner and more readable, as you don't have (often messy looking) conditional logic in the template. As long as it is named clearly then it is almost exactly the same a making it a component that conditionally returns an element, a pattern that I use elsewhere in this PR. 

I'd be interested in what @svillafe and @Ap0c think about this, as they probably have encountered this before and will have a preferred style

Reply:
yep fair enough, probably need a codebase-wide style decision on this one (I got away with a variation on the inline style here: https://github.com/guardian/support-frontend/blob/master/assets/components/bundle/bundle.jsx#L37-L38)